packages/manager/apps/web/client/app/private-database/order/clouddb/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,63 @@
+{
+  "private_database_order_clouddb_title": "Commander une base de données CloudDB",
+  "private_database_order_clouddb_subtitle": "L’offre CloudDB nécessite un service ayant accès au réseau internet public.",
+
+  "private_database_order_clouddb_database_type": "Type de base de données",
+  "private_database_order_clouddb_database_question": "Quel offre de base de données voulez-vous ?",
+  "private_database_order_clouddb_offer_title": "CloudDB",
+  "private_database_order_clouddb_offer_subtitle" : "Idéales en complément de tous nos produits Cloud OVH comme nos VPS, serveurs dédiés, instances Public Cloud et Web Hosting Cloud Web",
+
+  "private_database_order_clouddb_customization": "Personnalisation",
+  "private_database_order_clouddb_server_version": "Version du serveur",
+  "private_database_order_clouddb_server_version_placeholder": "Sélectionnez la version du serveur",
+  "private_database_order_clouddb_server_version_mariadb_101": "MariaDB 10.1",
+  "private_database_order_clouddb_server_version_mariadb_102": "MariaDB 10.2",
```

Comment:
So we're going to add translations for `MariaDB 10.3` and so on ?

Reply:
Sure it can be done in a better way ;)